tracing/src/test/java/org/mule/test/components/tracing/AbstractOpenTelemetryTracingTestCase.java

```diff
@@ -102,10 +100,12 @@ public AbstractOpenTelemetryTracingTestCase(String exporterType, String schema,
     this.port = port;
     this.path = path;
     this.secure = secure;
+
+    configureCollector();
```

Comment:
Instead of the constructor can we put in in an overridden doSetUpBeforeMuleContextCreation?